jena-base/src/main/java/org/apache/jena/atlas/lib/CacheFactory.java

```diff
@@ -43,7 +43,7 @@
 
     /** One slot cache */
     public static <Key, Value> Cache<Key, Value> createOneSlotCache() {
-        return new Cache1<>() ;
+        return new CacheGuava<>(1) ;
     }
```

Comment:
It's not used (if it is used elsewhere, then as it's not API, reasonable change is OK). The cache interface is useful though and `Supplier` does not cover the `getOrFill` pattern.  It also allows a switch from `Cache1` to a bigger, better cache as the interface is the same.